test/integration/crud_test.go

```diff
@@ -20,8 +20,15 @@ import (
 	"fmt"
 	"testing"
 
+	apiv1 "k8s.io/api/core/v1"
```

Comment:
kube imports should be grouped with those starting on line 31 (and that group sorted).